examples/interaction/tools/position_toolbar_clash.py

```diff
@@ -1,7 +1,17 @@
+''' A basic scatter plot with the toolbar positioned above the plot.
+This example demonstrates a way to set the toolbar's location relative
+to the edge of the plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.toolbar_location, bokeh.plotting.figure.scatter, bokeh.plotting.show
+    :refs: :ref:`_ug_interaction_tools_toolbar`
+    :keywords: position, location, toolbar
+'''
+
 from bokeh.plotting import figure, show
 
 p = figure(width=400, height=400,
-           title=None, toolbar_location="below")
+           title='Toolbar Location', toolbar_location="above")
```

Comment:
@huijeong1015 I think the location was set to below for a reason, in order to demonstrate how things look when a toolbar and and axis are both on the same side. I would recommend changing that back to "below" (and updating the comment accordingly)